rotkehlchen/tests/unit/decoders/test_aave_v3.py

```diff
@@ -1373,3 +1383,69 @@ def test_aave_v3_interest_on_transfer(ethereum_inquirer, ethereum_accounts) -> N
         ),
     ]
     assert events == expected_events
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('ethereum_accounts', [['0x4F243B4b795502AA5Cf562cB42EccD444c0321b0']])
+def test_aave_v3_lido_pool(
+        ethereum_inquirer: EthereumInquirer,
+        ethereum_accounts: list[ChecksumEvmAddress],
+) -> None:
+    tx_hash = deserialize_evm_tx_hash('0xd1e4360f6f7fb34f0f169aeb5e7d0ac0f025e3fed90903a50ee7b1bb694ac8d0')  # noqa: E501
+    get_or_create_evm_token(
+        evm_inquirer=ethereum_inquirer,
+        userdb=ethereum_inquirer.database,
+        evm_address=deserialize_evm_address('0x18eFE565A5373f430e2F809b97De30335B3ad96A'),
+        chain_id=ethereum_inquirer.chain_id,
+        protocol=CPT_AAVE_V3,
+        token_kind=EvmTokenKind.ERC20,
+        underlying_tokens=[UnderlyingToken(
+            address=string_to_evm_address('0x40D16FC0246aD3160Ccc09B8D0D3A2cD28aE6C2f'),
+            token_kind=EvmTokenKind.ERC20,
+            weight=ONE,
+        )],
+    )
+    events, _ = get_decoded_events_of_transaction(evm_inquirer=ethereum_inquirer, tx_hash=tx_hash)
+    borrow_amount, gas_fees, timestamp = '59000', '0.00170195956638183', TimestampMS(1734912023000)
+    expected_events = [
+        EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=0,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(amount=FVal(gas_fees)),
+            location_label=ethereum_accounts[0],
+            notes=f'Burn {gas_fees} ETH for gas',
+            counterparty=CPT_GAS,
+        ), EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=185,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.RECEIVE,
+            event_subtype=HistoryEventSubType.RECEIVE_WRAPPED,
+            asset=EvmToken('eip155:1/erc20:0x18577F0f4A0B2Ee6F4048dB51c7acd8699F97DB8'),
+            balance=Balance(amount=FVal(borrow_amount)),
+            location_label=ethereum_accounts[0],
+            notes=f'Receive {borrow_amount} variableDebtEthLidoGHO from AAVE v3',
```

Comment:
What is this example exactly? Borrowing 2 different tokens?

Reply:
yes, the new ones. Added it to verify that the lido tokens work as expected